langchain4j-open-ai/src/main/java/dev/langchain4j/model/openai/InternalOpenAiHelper.java

```diff
@@ -206,14 +208,17 @@ private static dev.ai4j.openai4j.chat.Parameters toOpenAiParameters(ToolParamete
 
     public static AiMessage aiMessageFrom(ChatCompletionResponse response) {
         AssistantMessage assistantMessage = response.choices().get(0).message();
+        String text = assistantMessage.content();
 
         List<ToolCall> toolCalls = assistantMessage.toolCalls();
         if (!isNullOrEmpty(toolCalls)) {
             List<ToolExecutionRequest> toolExecutionRequests = toolCalls.stream()
                     .filter(toolCall -> toolCall.type() == FUNCTION)
                     .map(InternalOpenAiHelper::toToolExecutionRequest)
                     .collect(toList());
-            return aiMessage(toolExecutionRequests);
+            return isNullOrEmpty(text) ?
+                    aiMessage(toolExecutionRequests) :
+                    new AiMessage(text, toolExecutionRequests);
```

Comment:
maybe we can add new static constructor method `aiMessage(String, List)` to follow the same style as in previous line?